cmd/network/verification.go

```diff
@@ -218,6 +219,12 @@ func (e *EgressVerification) setup(ctx context.Context) (*aws.Config, error) {
 		e.log.Debug(ctx, "cluster %s found from OCM: %s", e.ClusterId, cluster.ID())
 		e.cluster = cluster
 
+		if e.cluster.Hypershift().Enabled() {
+			return nil, errors.New("failed to run network verifier." +
+				" Hosted control plane clusters are not supported." +
+				" Instead, please use the network verifier script in ops-sop/hypershift/utils/verify-egress.sh")
+		}
```

Comment:
If you agree, I think it would be better to print a warning message that SRE has to confirm to bypass. Something like "Generaly, SRE has insufficient AWS permissions to run network verifier on hosted control plane clusters, run anyway?"